get_involved.md

```diff
@@ -5,6 +5,20 @@ permalink: /get_involved/
 toc: false
 ---
 
+## Upcoming Events
+
+**Bug Triage** - every Tuesday and Friday in IRC. Come and participate in real-time. See the
+[triage process](http://docs.pulpproject.org/dev-guide/contributing/bugs.html#triage-process) for
+more details. You can also read through the [triage archives](https://pulpadmin.fedorapeople.org/triage/pulp-dev/?C=N;O=D)
+
+**Community Demo** - Wednesday, April 10, 2019 at [15:00 UTC](https://www.worldtimebuddy.com/?qm=1&lid=100,4487042,3078610&h=100&date=2019-4-10&sln=15-16)
```

Comment:
@CodeHeeler FYI this would be another place to adverise our demos when they schedule/change